### PR TITLE
Additional CSS Nudge: Premium Plan -> Explorer Plan

### PR DESCRIPTION
**Pull request by Aurorum**
## Proposed changes:

* Uses the term "Explorer plan" instead of "Premium Plan" in the CSS nudge (plan names should no longer be capitalised, as happens in the rest of Calypso) 

## Jetpack product discussion

N/A.

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

Absolutely no idea if this works for certain - I don't think that I can test changes on WP.com! But it seems logical based on the code that this should work.

You can test this under "Additional CSS" in the Customizer on a Free/Starter plan site.

Screenshot 2024-02-06 at 22 41 08

### CODE REVIEW

**Review by jeherve — CHANGES_REQUESTED** [review]
Thanks for taking care of that. 

This would be a good opportunity to remove the changes that were added to that file in #26362. The `ENABLE_PRO_PLAN` constant is no longer available, and since you're updating the `$nudge_text` string the `preg_replace` that follows can no longer happen.

Do you think you could remove that logic?

Thanks!

**Comment by Aurorum** [comment]
All done for this file! But out of interest, where's this line being shown? I can't find it when trying to do so on WP.com. Besides the outdated plan name, surely it should be prompting an upgrade to Premium/Explorer rather than Business? 

https://github.com/Automattic/jetpack/blob/9b0d6d052853ea7e1d5457bcf50d31d27dad5c9e/projects/plugins/jetpack/modules/masterbar/nudges/additional-css/class-atomic-additional-css-manager.php#L42

File: projects/plugins/jetpack/modules/masterbar/nudges/additional-css/class-css-nudge-customize-control.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ class CSS_Nudge_Customize_Control extends \WP_Customize_Control {
 	public function render_content() {
 		$cta_url           = $this->cta_url;
 		$nudge_copy        = $this->nudge_copy;
-		$nudge_button_copy = __( 'Upgrade Now', 'jetpack' );
+		$nudge_button_copy = __( 'Upgrade now', 'jetpack' );
```
Comment (Aurorum) [inline projects/plugins/jetpack/modules/masterbar/nudges/additional-css/class-css-nudge-customize-control.php:45]: This is very minor but the preference is now title case and that's what happens with all the other buttons across WP.com/Jetpack. I can switch back if it's an intentional exception, but I suspect this was just never changed when the other buttons were (see Automattic/wp-calypso#38231).

**Comment by jeherve** [comment]
> out of interest, where's this line being shown? I can't find it when trying to do so on WP.com. Besides the outdated plan name, surely it should be prompting an upgrade to Premium/Explorer rather than Business?

Good catch. It's not something that's easy to reproduce in the wild. It is only displayed on WoA sites that do not have a Business plan. You could update that string to "Creator" now! 👍

File: projects/plugins/jetpack/modules/masterbar/nudges/additional-css/class-css-nudge-customize-control.php (hunk 1 above)
Comment (jeherve) [reply projects/plugins/jetpack/modules/masterbar/nudges/additional-css/class-css-nudge-customize-control.php:45]: This is a good idea, good catch!

**Comment by Aurorum** [comment]
All done! But slightly curious how you can have a non-Business plan WoA site? I didn't think that combination was possible unless a Business plan expired (in which case, shouldn't it still be an Explorer feature?).

**Comment by jeherve** [comment]
> But slightly curious how you can have a non-Business plan WoA site?

You cannot really get one. This is something that is still under development. :)

**Review by jeherve — APPROVED** [review]
Looking good now, thank you!

**Pull request merged**